superset/config.py

```diff
@@ -188,7 +188,9 @@
 # ---------------------------------------------------
 # Feature flags that are on by default go here. Their
 # values can be overridden by those in super_config.py
-FEATURE_FLAGS = {}
+FEATURE_FLAGS = {
+    'IS_EPOCH_S_TRULY_UTC': False,
```

Comment:
@xtinec , please review this use of the feature flags.

Reply:
Would be great to have a description of the feature flag here as a comment. Also, @xtinec how is one supposed to set a single entry in the feature flag in `superset_config.py`? Seems like someone would have to either override all of the `FEATURE_FLAGS` dict or none.

Reply:
@xtinec said she would be fixing this behavior. Basically the proper merging needs to be done. For now, we can work around it by assuming the default value in the code proper.

Reply:
@mistercrunch, is there anything else you would like me to improve in this PR ? Thanks

Reply:
@agrawaldevesh @mistercrunch I have a PR coming up shortly to allow merging of feature flags in `superset_config.py` and those set by default in `config.py`.